### PR TITLE
No search index for wip docs

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -24,6 +24,10 @@ RewriteRule ^(.*)$      $1.html [END]
   Header set Access-Control-Allow-Headers "Content-Type"
 </If>
 
+# No search index for wip docs: https://developers.google.com/search/reference/robots_meta_tag
+<If "%{REQUEST_URI} =~ m#^/platform-guide/wip/(.*)$#">
```
Comment (patriknw) [inline .htaccess:28]: pls review that I got the syntax right here

Comment (octonato) [reply .htaccess:28]: Hard to say if it's correct without being able to test it. Looks good, but 🤷

Comment (jrudolph) [reply .htaccess:28]: Easy enough to verify with curl once merged. Whether google/other crawlers actually pick it up is then another question... (which can be verified using the search console).

**Review by octonato — APPROVED** [review]

File: .htaccess (hunk 1 above)
Comment (octonato) [reply .htaccess:28]: let's merge and check then

**Pull request merged**